setup.py

```diff
@@ -28,7 +28,7 @@ def finalize_options(self):
 else:
     pass  # don't include binaries; user will have to arrange for the lib
 
-runtime_deps = ["cffi>=1.10"]
+runtime_deps = ["cffi>=1.10", "rubicon-objc>=0.4.1"]
```

Comment:
This should make the dependency for osx only.
```suggestion
runtime_deps = ["cffi>=1.10", "rubicon-objc>=0.4.1; platform_system=='Darwin'"]
```

Reply:
I'm pretty sure that should be `sys_platform == 'darwin'`

Reply:
I think both are fine :)
See https://docs.python.org/3/library/platform.html#platform.system and https://www.python.org/dev/peps/pep-0508/#environment-markers

Reply:
But if you've used `sys_platform` in another project, I would feel safer using that, haha.

Reply:
I am unsure!

Reply:
I'm pretty sure both should work. Should be easy enough to test.

Reply:
I just tested this line. If I run `pip install .` with exactly this line, then `rubicon-objc` is not installed. If I replace "Darwin" with "Windows" then it does.

Reply:
> I don't think we can make the installation happen with those flags via `install_requires` in our `setup.py`, or can we? And even if we could, I don't think our `setup.py` would be the correct place for such a fix.

That's correct, you would need to instruct users to install CFFI like that manually via the README. Sadly.

Reply:
@marcdownie if we add this change, than this PR is good to go, I think!